apps/commudle-admin/src/app/feature-modules/community-control-panel/components/admin-community-hackathon/admin-community-hackathon.component.ts

```diff
@@ -1,21 +1,38 @@
 import { Component, OnDestroy, OnInit } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { Subscription } from 'rxjs';
 import { SeoService } from 'apps/shared-services/seo.service';
+
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service